rpc_jobs/install_jenkins_plugins.yml

```diff
@@ -19,12 +19,11 @@
             common = load 'pipeline_steps/common.groovy'
             common.install_ansible()
             sh """#!/bin/bash
-              . .venv/bin/activate
+              . ../.venv/bin/activate
```

Comment:
This is in the top level job, which makes the current dir easy to determine, so relative paths can be used. When working in a lib function you don't know if the caller is within a dir() block so absolute paths are better.